tap_stripe/__init__.py

```diff
@@ -547,7 +556,7 @@ def convert_dict_to_stripe_object(record):
 
 # pylint: disable=too-many-locals
 # pylint: disable=too-many-statements
-def sync_stream(stream_name):
+def sync_stream(stream_name, is_sub_stream=False):
```

Comment:
Explain variable names in the function comments.

Reply:
Added the details of the var name in function comments